src/components/GameSection/GameSection.js

```diff
@@ -28,6 +28,10 @@ const GameSection = (category, handleUserAnswer) => {
       : TextQuestion(question.questionObject);
   const categoryQuestion = TextQuestion(RULES[category].question);
 
+  const categoryQuestionWithTimer = document.createElement('div');
+  categoryQuestionWithTimer.classList.add('gameSection__questionAndTimer');
+  categoryQuestionWithTimer.append(categoryQuestion, gameSection.timer);
```

Comment:
Dlaczego to połączyłeś?

Reply:
Bo według figmy timer jest na prawo od tego pytania. A że wykorzystujemy do stylowania między innymi flexa, to żeby były koło siebie dodałem je do jednego kontenera.